target/linux/ath79/image/generic-ubnt.mk

```diff
@@ -130,6 +130,14 @@ define Device/ubnt_bullet-m-xw
 endef
 TARGET_DEVICES += ubnt_bullet-m-xw
 
+define Device/ubnt_edgeswitch-8xp
+  $(Device/ubnt-sw)
+  DEVICE_MODEL := EdgeSwitch-8XP
```

Comment:
Space instead of hyphen to be consistent with model.
If there are no other EdgeSwitch 8XP variants, you should remove the DEVICE_VARIANT in Device/ubnt-sw, because otherwise that will be added to the device title.